pandas/tests/test_graphics.py

```diff
@@ -1065,12 +1065,6 @@ def test_implicit_label(self):
         self._check_text_labels(ax.xaxis.get_label(), 'a')
 
     @slow
-    def test_explicit_label(self):
```

Comment:
It is indeed possible people rely on this, but it *is* very inconsistent, so strange it is put into a test. `label` clearly should not alter the 'xlabel', as it is used in matplotlib also for the legends